code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+	origin_tech = "biotech=5"
+
+/obj/item/organ/internal/heart/gland/heals/weak/activate()
+	to_chat(owner, "<span class='notice'>You feel slightly revitalized.</span>")
+	owner.adjustToxLoss(-10)
+	owner.adjustBruteLoss(-10)
+	owner.adjustOxyLoss(-10)
+	owner.adjustFireLoss(-10)
+
+/obj/item/ammo_casing/c10mm/spent
+	projectile_type = null
+
+/obj/item/ammo_casing/c45/spent
+	projectile_type = null
+
+////// MACHINARY //////
+
+/obj/machinery/computer/id_upgrader/dsbssigma
+	icon_keyboard = "laptop_key"
+	access_to_give = list(ACCESS_AWAY01)
+
+/obj/machinery/vending/medical/syndicate_access/dsbssigma
+	req_access = list(ACCESS_AWAY01)
+	products = list(/obj/item/reagent_containers/syringe = 4, /obj/item/reagent_containers/food/pill/patch/styptic = 2, /obj/item/reagent_containers/food/pill/patch/silver_sulf = 2, /obj/item/reagent_containers/applicator/brute = 1, /obj/item/reagent_containers/applicator/burn = 1,
+					/obj/item/reagent_containers/glass/bottle/charcoal = 1, /obj/item/reagent_containers/glass/bottle/salicylic = 1, /obj/item/reagent_containers/glass/bottle/saline = 1, /obj/item/reagent_containers/glass/bottle/ether = 1,
+					/obj/item/reagent_containers/glass/bottle/atropine = 1, /obj/item/reagent_containers/glass/bottle/oculine = 1, /obj/item/reagent_containers/glass/bottle/toxin = 1, /obj/item/reagent_containers/syringe/antiviral = 2,
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = 1
+
+/obj/machinery/light/broken
+	active_power_usage = 0
+	icon_state = "tube-broken"
+	status = 2
+
+/obj/machinery/light/small/broken
+	active_power_usage = 0
+	icon_state = "bulb-broken"
+	status = 2
+
+/obj/machinery/door/poddoor/shutters/rusted
+	name = "Rusted Shutter"
+	desc = "Heavy duty shutters that has been warped and corroded by the elements. It does not seem it will take much to break it down by force."
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 15, "energy" = 15, "bomb" = 0, "bio" = 50, "rad" = 100, "fire" = 50, "acid" = 25)
+	damage_deflection = 0
+	max_integrity = 300
+	icon_state = "rusted_closed"
+
+/obj/machinery/door/poddoor/shutters/rusted/do_animate(animation)
+	switch(animation)
+		if("opening")
+			flick("rusted_opening", src)
```

Comment:
This proc is the same as it's parent `/obj/machinery/door/poddoor` except for the flick values.
Can you make these into variables that can be changed on the object?